src/components/TwitchSidebar/_twitch-sidebar.scss

```diff
@@ -0,0 +1,17 @@
+.twitch-stream {
+    display: flex;
+    // justify-content: space-between;
+    align-items: center;
+    width:100%;
+    a {
+        color:inherit;
+        &:hover {
+        text-decoration: none !important;
```

Comment:
I would find a way to be more specific with your css selector so you dont need to use `!important`. If you get stuck on that let me know it can be tricky sometimes.

Reply:
for the life of me I can't select this another way without using important to get the underline to stop